From 93297379d861b4c152debb828d179ed273f9b26c Mon Sep 17 00:00:00 2001
From: phlmm <milev.philip@gmail.com>
Date: Sat, 23 Oct 2021 13:42:15 +0000
Subject: [PATCH] FindLibDvd.cmake: fix logic for reusing system library

---
 cmake/modules/FindLibDvd.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/modules/FindLibDvd.cmake b/cmake/modules/FindLibDvd.cmake
index 58a7aab309..ebfa9e3140 100644
--- a/cmake/modules/FindLibDvd.cmake
+++ b/cmake/modules/FindLibDvd.cmake
@@ -1,4 +1,4 @@
-if(KODI_DEPENDSBUILD)
+if(NOT KODI_DEPENDSBUILD)
   set(_dvdlibs dvdread dvdnav)
   set(_handlevars LIBDVD_INCLUDE_DIRS DVDREAD_LIBRARY DVDNAV_LIBRARY)
   if(ENABLE_DVDCSS)
